samples/subscribing-to-streams/Program.cs

```diff
@@ -52,7 +53,7 @@ class Program {
 			var checkpoint = StreamPosition.Start;
 			await client.SubscribeToStreamAsync(
 				"some-stream",
-				checkpoint,
+				SubscriptionStreamPosition.After(checkpoint),
```

Comment:
probably want something along the lines of the $all equivalent, we're still skipping the first event here

Reply:
sample has been updated